integration/helper_test.go

```diff
@@ -263,6 +263,180 @@ options:
 			Expect(err).ToNot(HaveOccurred())
 			assertNoErrors()
 		})
+		It("gpbackup_helper will not error out when plugin writes something to stderr", func() {
+			setupRestoreFiles("", true)
+
+			err := exec.Command("touch", "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR").Run()
+			Expect(err).ToNot(HaveOccurred())
+
+			args := []string{
+				"--toc-file", tocFile,
+				"--oid-file", restoreOidFile,
+				"--pipe-file", pipeFile,
+				"--content", "1",
+				"--single-data-file",
+				"--restore-agent",
+				"--data-file", dataFileFullPath,
+				"--plugin-config", examplePluginTestConfig}
+			helperCmd := exec.Command(gpbackupHelperPath, args...)
+
+			var outBuffer bytes.Buffer
+			helperCmd.Stdout = &outBuffer
```

Comment:
Does test pass even if plugin writes something to stdout instead stderr?

Reply:
The helper will examine only stderr of the plugin to check for warnings. Here we check the output of the gpbackup itself. It is a replacement for `CombinedOutput()` from the original patch (which now we can't use since we got rid of the goroutines)

Reply:
Why can't we use CombinedOutput here?